### DISCUSSION TITLE
UK - Anyone having Issue with Scan?

### DISCUSSION THREAD

**Discussion by thetallchap** [jef/streetmerchant#2123] [Help]
Is anyone else having issues just with Scan in the UK? I have no other issues (apart from CloudFlare on CCL which i understand you can't get around) but on Scan i see no results. 

If i limit the run to just Scan i get the 'running linksbuilder...' message but then nothing.

I guessing i'm missing something obvious... ;) many thanks

**Comment by damiankrzyz** [comment]
same for me
